ibm/mas_devops/roles/sls_install/tasks/main.yml

```diff
@@ -3,12 +3,12 @@
 # 1. Check for missing properties that do not have defaults
 # -----------------------------------------------------------------------------
 - name: "Fail if sls_entitlement_username has not been provided"
-  when: sls_entitlement_username is not defined or sls_entitlement_username == ""
+  when: sls_entitlement_username is not defined or sls_entitlement_username|length == 0
```

Comment:
What's the reason behind this change?  Also, while updating these checks please convert them to asserts while doing so (see #124)